models/op/retropgf/optimism/op_retropgf_optimism_round2_recipients.sql

```diff
@@ -22,8 +22,8 @@ with attestations as (
 SELECT 
     nm.block_date, nm.recipient as submitter_address, nm.issuer
     , trim(nm.val_string) AS recipient_name, trim(ca.val_string) AS recipient_category
-    , cast( regexp_replace(aw.val_string, '[^0-9\\.]+', '') AS double ) AS award_amount
-    , '{{op_token}}' AS award_token
+    , cast( regexp_replace(aw.val_string, '[^0-9\\.]+', '') AS uint256 ) AS award_amount
```

Comment:
i didn't have the full context, but i'm guessing the cast to double needs to be a larger int datatype -- will `uint256` work? does that make sense for the column? i guess CI will output a potential failure shortly if not

Reply:
okay, looks like i picked the wrong spot for the error. i can revert this change